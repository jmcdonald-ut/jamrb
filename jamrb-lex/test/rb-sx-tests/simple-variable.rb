x = 4
y = 3
z = x + y

puts "#{x} + #{y} = #{z}"
