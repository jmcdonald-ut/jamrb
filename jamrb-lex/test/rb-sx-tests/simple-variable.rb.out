((Token ((1 0) on_ident "x"))
 (Token ((1 1) on_sp " "))
 (Token ((1 2) on_op "="))
 (Token ((1 3) on_sp " "))
 (Token ((1 4) on_int "4"))
 (Token ((1 5) on_nl "\\n"))
 (Token ((2 0) on_ident "y"))
 (Token ((2 1) on_sp " "))
 (Token ((2 2) on_op "="))
 (Token ((2 3) on_sp " "))
 (Token ((2 4) on_int "3"))
 (Token ((2 5) on_nl "\\n"))
 (Token ((3 0) on_ident "z"))
 (Token ((3 1) on_sp " "))
 (Token ((3 2) on_op "="))
 (Token ((3 3) on_sp " "))
 (Token ((3 4) on_ident "x"))
 (Token ((3 5) on_sp " "))
 (Token ((3 6) on_op "+"))
 (Token ((3 7) on_sp " "))
 (Token ((3 8) on_ident "y"))
 (Token ((3 9) on_nl "\\n"))
 (Token ((4 0) on_ignored_nl "\\n"))
 (Token ((5 0) on_ident "puts"))
 (Token ((5 4) on_sp " "))
 (Token ((5 5) on_tstring_beg "\\\""))
 (Token ((5 6) on_embexpr_beg "\\#{"))
 (Token ((5 8) on_ident "x"))
 (Token ((5 9) on_embexpr_end "}"))
 (Token ((5 10) on_tstring_content " + "))
 (Token ((5 13) on_embexpr_beg "\\#{"))
 (Token ((5 15) on_ident "y"))
 (Token ((5 16) on_embexpr_end "}"))
 (Token ((5 17) on_tstring_content " = "))
 (Token ((5 20) on_embexpr_beg "\\#{"))
 (Token ((5 22) on_ident "z"))
 (Token ((5 23) on_embexpr_end "}"))
 (Token ((5 24) on_tstring_end "\\\""))
 (Token ((5 25) on_nl "\\n")))
